src/frontend/src/components/ExternalUserGame.js

```diff
@@ -2,14 +2,18 @@ import React, { useEffect, useState } from "react";
 // components
 import ExternalUserGrid from "./ExternalUserGrid";
 
-export default function ExternalUserGame({ externalUserGameGuesses }) {
+export default function ExternalUserGame({ externalUserGameGuesses, userName }) {
   const [corrections, setCorrections] = useState([])
+  const [turnCount, setTurnCount] = useState(0)
 
   useEffect(() => {
     if (externalUserGameGuesses != null) {
       const newCorrections = []
+      let turnCounter = 0
       // Transform the guesses array into a format that the FE can understand
       for (let turn of externalUserGameGuesses) {
+        if(turn === "") continue
```

Comment:
im guessing this occurs if we ever load a external user and try accessing the turn when its not done rendering?

Reply:
This was to prevent the weird bug from happening where the users would correction would look like this and then the turn number would be off. 
```
{
    "google-oauth2|115680850015191286676": [
        "",
        "",
        "XYYXY",
        "XXYYX"
    ],
    "google-oauth2|117278212575785015953": [
        "",
        "",
        "XXXXY",
        "XXXXY"
    ]
}
```